src/main/java/com/cloudant/http/interceptors/BasicAuthInterceptor.java

```diff
@@ -41,7 +41,7 @@ public BasicAuthInterceptor(String userinfo, String authHeader) {
     @Override
     public HttpConnectionInterceptorContext interceptRequest(HttpConnectionInterceptorContext
                                                                      context) {
-        context.connection.requestProperties.put("Authorization", String.format("Basic %s",
+        context.connection.requestProperties.put(authHeader, String.format("Basic %s",
```

Comment:
I know that the `authHeader` field wasn't added by this PR - but why would you ever want to change the value of this string?